server/norace_test.go

```diff
@@ -3435,26 +3430,34 @@ func TestNoRaceJetStreamClusterCorruptWAL(t *testing.T) {
 
 	cl = c.consumerLeader("$G", "TEST", "dlc")
 	mset, err = cl.GlobalAccount().lookupStream("TEST")
-	if err != nil {
-		t.Fatalf("Unexpected error: %v", err)
-	}
+	require_NoError(t, err)
 	o = mset.lookupConsumer("dlc")
-	if o == nil {
-		t.Fatalf("Error looking up consumer %q", "dlc")
-	}
+	require_NoError(t, err)
+
 	// Grab underlying raft node and the WAL (filestore) and truncate it.
 	// This will simulate the WAL losing state due to truncate and we want to make sure it recovers.
+
 	fs = o.raftNode().(*raft).wal.(*fileStore)
 	state = fs.State()
-	fs.Truncate(state.FirstSeq)
+	err = fs.Truncate(state.FirstSeq)
+	require_NoError(t, err)
+	state = fs.State()
 
 	sub, err = js.PullSubscribe("foo", "dlc")
-	if err != nil {
-		t.Fatalf("Unexpected error: %v", err)
-	}
+	require_NoError(t, err)
 
 	// This will cause us to stepdown and truncate our WAL.
-	sub.Fetch(100)
+	msgs, err := sub.Fetch(100)
```

Comment:
I remember removing the error check in the past because it would fail there often, and based on the comment above, this call would cause the stepdown, etc.. so I figure that fetching may fail anyway. The actual check of getting messages was done below.

Reply:
The fetch would fail before with old logic that we could not fill the batch, but now we should get something all the time, which I tested. Mostly get all 100, sometimes were getting partials which is why the checks on the consumer info state would fail for those.

Reply:
You were correct, ignore error and made fix to raft layer.